src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -8,8 +8,8 @@ import { matchRouteWithPattern } from "~/src/util/match-route-with-pattern";
 import { fetchAndSaveUserJiraAdminStatus } from "middleware/jira-admin-permission-middleware";
 
 export const jiraSymmetricJwtMiddleware = async (req: Request, res: Response, next: NextFunction) => {
-
-	const token = req.query?.["jwt"] || req.cookies?.["jwt"] || req.body?.["jwt"];
+	const authHeader = req.headers["authorization"];
+	const token = req.query?.["jwt"] || req.cookies?.["jwt"] || req.body?.["jwt"] || authHeader?.substring(7);
```

Comment:
When the request comes in from Jira, the JWT with be passed in the auth header so we need to check there as well now.